release_notes/gitops-release-notes-1-16.adoc

```diff
@@ -0,0 +1,36 @@
+//OpenShift GitOps Release Notes
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="gitops-release-notes"]
+= {gitops-title} release notes
+:context: gitops-release-notes
+
+toc::[]
+
+[NOTE]
+====
+For additional information about the OpenShift {gitops-shortname} lifecycle and supported platforms, refer to the link:https://access.redhat.com/support/policy/updates/openshift_operators[OpenShift Operator Life Cycles] and link:https://access.redhat.com/support/policy/updates/openshift[Red{nbsp}Hat {OCP} Life Cycle Policy].
+====
+
+Release notes contain information about new and deprecated features, breaking changes, and known issues. The following release notes apply for the most recent OpenShift {gitops-shortname} releases on {OCP}.
+
+{gitops-title} is a declarative way to implement continuous deployment for cloud native applications. {gitops-title} ensures consistency in applications when you deploy them to different clusters in different environments, such as development, staging, and production. {gitops-title} helps you automate the following tasks:
+
+* Ensure that the clusters have similar states for configuration, monitoring, and storage
+* Recover or recreate clusters from a known state
+* Apply or revert configuration changes to multiple {OCP} clusters
+* Associate templated configuration with different environments
+* Promote applications across clusters, from staging to production
+
+For an overview of {gitops-title}, see xref:../understanding_openshift_gitops/about-redhat-openshift-gitops.adoc#about-redhat-openshift-gitops[About Red Hat OpenShift GitOps].
```

Comment:
```suggestion
For an overview of {gitops-title}, see xref:../understanding_openshift_gitops/about-redhat-openshift-gitops.adoc#about-redhat-openshift-gitops[About Red{nbsp}Hat OpenShift GitOps].
```

- If possinle consider using an attribute for Red Hat Openshift Gitops

Reply:
Fixed.